### PR TITLE
NMS-14352: Update terminology

### PR DESCRIPTION
**Pull request by emarsh1995**
- Changed "surveillance category/categories" to "node tag/tags."
- Changed "node categories" to "node tags."
- Updated formatting.

Note: I couldn't find any instances of "provisioning group" or "node category" (singular) in the documentation, so those items haven't been changed.

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-14352

### CODE REVIEW

File: docs/modules/operation/pages/admin/webui/dashboard.adoc [hunk 1]
```diff
@@ -50,12 +49,13 @@ The following information are shown in:
...
-4. Count: Number of alarms deduplicated by the reduction key of the alarm
-5. Last Time: Time for the last occurrence of the alarm
-6. Log Msg: The log message from the event which is the source for this alarm. It is specified in the event configuration file in `<logmsg />`
+1. Node: Node label of the node the alarm is associated.
+2. Severity: Severity of the alarm.
+3. UEI: Shows the UEI of the alarm.
+4. Count: Number of alarms deduplicated by the reduction key of the alarm.
+5. Last Time: Time for the last occurrence of the alarm.
+6. Log Msg: The log message from the event which is the source for this alarm.
+It is specified in the event configuration file in `<logmsg />`
```
Comment (mmahacek) [inline docs/modules/operation/pages/admin/webui/dashboard.adoc:58]:
```suggestion
6. Log Msg: The log message from the event definition.
```

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 2]
```diff
@@ -34,19 +33,17 @@ For more details on how JasperReports works, please refer to the Jaspersoft Stud
...
-
 [options="header, autowidth"]
 |===
 | Report Name            | Category     | Description
 
 |  xref:operation:database-reports/templates/avail-summary.adoc[Availability Summary]
 | Status
-|Summary of statistics on outages and availability for user-specified surveillance category during a user-specified number of days.
-
+|Summary of statistics on outages and availability for user-specified node tag during a user-specified number of days.
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:42]:
```suggestion
| Summary of statistics on outages and availability for user-specified node tag during a user-specified number of days.
```

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 3]
```diff
@@ -86,7 +83,7 @@ It also shows how many will expire in the next 12 months.
 
 | xref:operation:database-reports/templates/response-summary.adoc[Response Time Summary]
 | Status
-| Summary of response times across surveillance category.
+| Summary of response times across node tag.
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:86]:
```suggestion
| Summary of response times for nodes with a given tag.
```

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 4]
```diff
@@ -387,5 +380,4 @@ An SSL-secured connection can be established even if `org.opennms.netmgt.jasper.
 == Limitations
 
 Only a JDBC Datasource to the {page-component-title} database connection can be passed to a report, or no datasource at all.
-One does not have to use the data source, though.
-
+One does not have to use the data source, though.
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:383]: Is this sentence necessary?

File: docs/modules/operation/pages/database-reports/templates/avail-summary.adoc [hunk 5]
```diff
@@ -1,18 +1,18 @@
 [[avail-summary-report]]
 = Availability Summary Report (Default Configuration for Past 7 Days)
 
-This report provides a summary of availability across one or more surveillance categories.
+This report provides a summary of availability across one or more node tags.
 
-Each surveillance category includes a table of average, minimum, and maximum values for the following availability data for the specified surveillance category over the specified date range:
+Each node tag includes a table of average, minimum, and maximum values for the following availability data for the specified node tag over the specified date range:
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/templates/avail-summary.adoc:6]:
```suggestion
For each node tag, the report includes a table of average, minimum, and maximum values for the following availability data over the specified date range:
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Looking good. Some changes.

File: docs/modules/operation/pages/database-reports/database.adoc (hunk 4 above)
Comment (Bonrob2) [reply docs/modules/operation/pages/database-reports/database.adoc:383]: Agreed, let's remove it.

Comment (emarsh1995) [reply docs/modules/operation/pages/database-reports/database.adoc:383]:
```suggestion

```

File: docs/modules/operation/pages/database-reports/templates/avail-summary.adoc [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
-* Number of outages
-* Mean Time To Repair ("MTTR") (in hours)
-* Length of time for each outage (in hours)
-* Percent outage (per unit in category)
-* Percent availability (per unit in category)
+* Number of outages.
+* Mean Time To Repair ("MTTR") (in hours).
+* Length of time for each outage (in hours).
+* Percent outage (per unit in category).
+* Percent availability (per unit in category).
```
[the remark this answers is not in the document]
Comment (emarsh1995) [reply docs/modules/operation/pages/database-reports/templates/avail-summary.adoc:12]: I believe it should be; I'll update this.

Comment (emarsh1995) [inline docs/modules/operation/pages/database-reports/templates/avail-summary.adoc:12]:
```suggestion
* Percent outage (per unit in tag).
* Percent availability (per unit in tag).
```

File: docs/modules/operation/pages/database-reports/templates/availability.adoc [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
-* Number of outages
-* Mean Time To Repair ("MTTR") (in hours)
-* Length of time for each outage (in hours)
-* Percent outage (per node in category)
-* Percent availability (per node in category)
+* Number of outages.
+* Mean Time To Repair ("MTTR") (in hours).
+* Length of time for each outage (in hours).
+* Percent outage (per node in category).
+* Percent availability (per node in category).
```
Comment (emarsh1995) [inline docs/modules/operation/pages/database-reports/templates/availability.adoc:11]:
```suggestion
* Percent outage (per node in tag).
* Percent availability (per node in tag).
```

File: docs/modules/operation/pages/admin/webui/dashboard.adoc [hunk 8]
```diff
@@ -88,13 +88,13 @@ To give an overview information about ongoing Outages in the network, the dashbo
 image::dashboard/05_dashboard-outages.png[]
 
 1. Node: Label of the monitored node with ongoing outages.
-2. Current Outages: Number of services on the node with outages and total number of monitored services, e.g. with the natural meaning of "3 of 3 services are affected".
+2. Current Outages: Number of services on the node with outages and total number of monitored services, e.g. with the natural meaning of "3 of 3 services are affected."
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply docs/modules/operation/pages/admin/webui/dashboard.adoc:90]: Looking at the sample graphic now, I think it's just a way to explain the absence of nouns in what is displayed. I think the phrase confuses things, so suggest the following:

```suggestion
2. Current Outages: Number of services on the node with outages out of the total number of monitored services; for example, "3 of 3 services are affected".
```

File: docs/modules/operation/pages/admin/webui/dashboard.adoc [hunk 9]
```diff
@@ -69,12 +69,12 @@ The dashboard should show outstanding notifications in the NOC to provide an ove
 .Information displayed in the Notifications component
 image::dashboard/04_dashboard-notifications.png[]
 
-1. Node: Label of the monitored node the notification is associated with
-2. Service: Name of the service the notification is associated with
-3. Message: Message of the notification
-4. Sent Time: Time when the notification was sent
-5. Responder: User name who acknowledged the notification
-6. Response Time: Time when the user acknowledged the notification
+1. Node: Label of the monitored node the notification is associated with.
```
[the remark this answers is not in the document]
Comment (emarsh1995) [reply docs/modules/operation/pages/admin/webui/dashboard.adoc:71]: I'd lean towards "monitored" being implied. @mmahacek, do you think we can just say "node" here?

[the remark this answers is not in the document]
Comment (mmahacek) [reply docs/modules/operation/pages/admin/webui/dashboard.adoc:71]: Totally.

**Review by Bonrob2 — COMMENTED** [review]
LGTM!

**Review by Bonrob2 — APPROVED** [review]
In previous comment, I clicked before I approved. :-)

**Review by mmahacek — APPROVED** [review]
No complaints here!

File: docs/modules/operation/pages/admin/webui/dashboard.adoc [hunk 10]
```diff
@@ -88,13 +88,13 @@ To give an overview information about ongoing Outages in the network, the dashbo
...
-2. Current Outages: Number of services on the node with outages and total number of monitored services, e.g. with the natural meaning of "3 of 3 services are affected".
+2. Current Outages: Number of services on the node with outages and total number of monitored services, e.g. with the natural meaning of "3 of 3 services are affected."
 3. 24 Hour Availability: Availability of all services provided by the node calculated by the last 24 hours.
 
 [[ga-dashboard-resource-graph-viewer]]
 === Resource Graph Viewer
 
-The Resource Graph Viewer lets you navigate to time series data reports to give a quick entry point to diagnose performance issues. 
+The Resource Graph Viewer lets you navigate to time series data reports to give a quick entry point to diagnose performance issues.
 This viewer filters nodes based on the Surveillance View context and shows one graph report at a time.
```
[the remark this answers is not in the document]
Comment (emarsh1995) [reply docs/modules/operation/pages/admin/webui/dashboard.adoc:97]: Cool - I think what we have should cover this. What do you think, @Bonrob2? :)

**Pull request merged**